.npmrc

```diff
@@ -0,0 +1 @@
+@gethue:registry=https://npm.pkg.github.com/
```

Comment:
This needed?

Reply:
Yes, but no bueno still, not sure what is missing. The idea is that this package is published on the GH npm, not the default one

Reply:
For now bueno, can look into it later.